cmd/sidecars/network-passt-binding/main.go

```diff
@@ -22,22 +22,28 @@ package main
 import (
 	"net"
 	"os"
+	"os/signal"
 	"path/filepath"
+	"syscall"
 
 	srv "kubevirt.io/kubevirt/cmd/sidecars/network-passt-binding/server"
 
 	"google.golang.org/grpc"
 
 	"kubevirt.io/kubevirt/pkg/hooks"
 	hooksInfo "kubevirt.io/kubevirt/pkg/hooks/info"
-	hooksV1alpha2 "kubevirt.io/kubevirt/pkg/hooks/v1alpha2"
+	hooksV1alpha3 "kubevirt.io/kubevirt/pkg/hooks/v1alpha3"
```

Comment:
The Shutdown API introduced in v1alpha3 is required. Otherwise, the sidecar container on the source pod remains stuck after migration.
It is ok the passt migration will be supported on new clusters only, this is a new 1.2 feature.
This code is not going to be backported to 1.1.

Reply:
> BTW, this is not related to the new migration support. If someone tries to run this sidecar on kubevirt v1.1 for example, it will not work.

Indeed. I'd suggest to keep v1alpha2 support in order to provide the functionality for a wider range of clusters even if we know that migration has problems. Note that migration should work as in the VM will be migrated but the state left in the source node is incomplete. Just a known bug that users can workaround it, probably.